src/main/java/de/zalando/tip/zalenium/util/CommonProxyUtilities.java

```diff
@@ -34,6 +35,29 @@ public static JsonElement readJSONFromUrl(String url) {
         return null;
     }
 
+    public static JsonElement readJSONFromFile(String fileName) {
+        try {
+            FileReader fr = new FileReader(new File(currentLocalPath(), fileName));
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Close this "FileReader". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2095)